packages/npm/financekit/src/index.tsx

```diff
@@ -21,9 +38,12 @@ const Financekit = NativeModules.Financekit
 
 const AuthorizationStatus = S.Literal('authorized', 'denied', 'notDetermined')
 
+export const uuid = S.UUID.pipe(S.brand('CNDL_UUID'))
+export const int = S.Int.pipe(S.brand('CNDL_INT'))
```

Comment:
```suggestion
export const S_CNDL_Int = S.Int.pipe(S.brand('CNDL_Int'))
```
same as above + in the brands I usually just add the `CNDL_` prefix to whatever it is, I don't uppercase everything